pulsar-client/src/main/java/com/yahoo/pulsar/client/impl/PartitionedConsumerImpl.java

```diff
@@ -142,7 +137,7 @@ private void receiveMessageFromConsumer(ConsumerImpl consumer) {
     }
 
     private void resumeReceivingFromPausedConsumersIfNeeded() {
-        if (incomingMessages.size() <= sharedQueueResumeThreshold && !pausedConsumers.isEmpty()) {
+        if (incomingMessages.size() < maxReceiverQueueSize && !pausedConsumers.isEmpty()) {
```

Comment:
I think the threshold was there to prevent from flickering for each message (block-unblock-block-unblock...). Would that be an issue here?